earth_enterprise/src/fusion/dbmanifest/dbmanifest_unittest.cpp

```diff
@@ -26,6 +26,22 @@
 #include "common/khxml/khdom.h"
 #include "fusion/dbmanifest/dbmanifest.h"
 
+struct MockGetIndextManifest : public DbManifest::IndextManifest {
+  std::vector<ManifestEntry> manifest_;
+  void operator()(geFilePool& /*filePool*/,
+                  const std::string& /*index_path*/,
+                  std::vector<ManifestEntry> &manifest,
+                  const std::string& /*tmpdir*/,
+                  const std::string& /*disconnect_prefix*/,
+                  const std::string& /*publish_prefix*/) {
+                    manifest = manifest_; // for now we keep things simple
+                  }
+};
+
+bool LessThan(const ManifestEntry& lhs_entry, const ManifestEntry& rhs_entry) {
+  return (lhs_entry.orig_path < rhs_entry.orig_path);
```

Comment:
we are comparing two paths as one is less than the other ?

Reply:
Yes.  I'm sorting the manifest entries based on original paths as strings.  The sort function uses less than for sorting so you have to provide a function that indicates which item is less than another item.  Based on how you want to sort things.